lib/scenic/adapters/postgres/views.rb

```diff
@@ -0,0 +1,56 @@
+module Scenic
+  module Adapters
+    class Postgres
+      # Fetches defined views from the postgres connection.
+      # @api private
+      class Views
+        def initialize(connection)
+          @connection = connection
+        end
+
+        # All of the views that this connection has defined.
+        #
+        # This will include materialized views if those are supported by the
+        # connection.
+        #
+        # @return [Array<Scenic::View>]
+        def all
+          views + materialized_views
```

Comment:
Can a nonmaterialized view depend on a materialized view? Is this going to get us into trouble like we saw previously with views depending on views that were alphabetically earlier than their dependancies?